lib/membrane_h264_format/h264.ex

```diff
@@ -84,6 +84,6 @@ defmodule Membrane.H264 do
           profile: profile_t()
         }
 
-  @enforce_keys [:width, :height, :framerate, :profile]
-  defstruct @enforce_keys ++ [alignment: :au, nalu_in_metadata?: false]
+  @enforce_keys [:width, :height, :profile]
+  defstruct @enforce_keys ++ [alignment: :au, nalu_in_metadata?: false, framerate: nil]
```

Comment:
if you put `framerate` at the beginning you can resign from `nil` as it's a default value

Reply:
Matter of taste, but I like the explicit approach :P

Reply:
I would stick to that more "explicit" approach :p